packages/fxa-content-server/app/styles/modules/_cad.scss

```diff
@@ -0,0 +1,3 @@
+#fxa-connected-heading {
+  background-image: image-url('circle-check.svg');
```

Comment:
Using scss for this because I don't think bg image paths in tailwind will be updated correctly in the content-server build process.  Happy to be wrong about this though.